FPM.ftd

```diff
@@ -59,6 +59,7 @@ email: arpita@fifthtry.com
   - Resumes: /featured/resumes/
   - Color Schemes: /featured/color-cs/
   - Component Libraries: /featured/components-library/
+  - How to Use Fonts: /featured/how-to-use-fonts/
```

Comment:
No, this folder is for featured stuff. This is a how to. It should go in the how-to folder.

Reply:
Make the url `fpm.dev/how-to/custom-fonts`.

Reply:
@amitu issue is fixed! Please, review again!

Reply:
The URL is not `fpm.dev/how-to/custom-fonts/`. The other document should have the URL: `fpm.dev/how-to/create-font-package/`. 

Move both the how-to documents in `how-to` folder.